### DISCUSSION TITLE
Error in deploying conductor

### DISCUSSION THREAD

**Discussion by k-rakshata** [Netflix/conductor#2191] [Q&A] [answered]
I am trying to deploy conductor on CentOS VM version 7. When I run the docker-compose build command, I am getting this error:

"ERROR: Service 'conductor-server' failed to build: The command '/bin/sh -c ./gradlew build -x test' returned a non-zero code: 1"

Has anybody else faced the same issue and got any resolution for this?

**Comment by maheshyaddanapudi** [comment, marked as answer]
Which version of Conductor are you using 2.X or 3.X ?

If its 3.x, then Java 11 is needed and the Dockerfile is still referring to the old Java 8 version. This causes a build failure.

I have forked the repository, updated the Dockerfile and submitted the PR. Once the PR is accepted, the issue should be fixed.

Until then please try using https://github.com/maheshyaddanapudi/conductor/blob/main/docker/server/Dockerfile for creating the Docker Image for Conductor Server

**Reply by k-rakshata** [reply]
I have updated the Dockerfile and changed the java version to 11, but now I am getting this error :

Step 5/16 : RUN ./gradlew build -x test
 ---> Running in 9fae60082d57
/usr/bin/env: ‘sh\r’: No such file or directory
ERROR: Service 'conductor-server' failed to build: The command '/bin/sh -c ./gradlew build -x test' returned a non-zero code: 127

I have added the required proxies. Can you please help me with this>

**Reply by maheshyaddanapudi** [reply]
Created a PR https://github.com/Netflix/conductor/pull/2206

**Reply by maheshyaddanapudi** [reply]
> I have updated the Dockerfile and changed the java version to 11, but now I am getting this error :
> 
> Step 5/16 : RUN ./gradlew build -x test
> ---> Running in 9fae60082d57
> /usr/bin/env: ‘sh\r’: No such file or directory
> ERROR: Service 'conductor-server' failed to build: The command '/bin/sh -c ./gradlew build -x test' returned a non-zero code: 127
> 
> I have added the required proxies. Can you please help me with this>

Please try changing the startup.sh last line conductor-server-\*-**all**.jar to conductor-server-\*-**boot**.jar
